pkg/operator/configobservation/network/observe_network.go

```diff
@@ -73,30 +75,56 @@ func ObserveRestrictedCIDRs(genericListers configobserver.Listers, recorder even
 }
 
 // ObserveServicesSubnet watches the network configuration and generates the
-// servicesSubnet
+// servicesSubnet (and bindAddress)
 func ObserveServicesSubnet(genericListers configobserver.Listers, recorder events.Recorder, existingConfig map[string]interface{}) (map[string]interface{}, []error) {
 	listers := genericListers.(configobservation.Listers)
 
 	out := map[string]interface{}{}
-	configPath := []string{"servicesSubnet"}
-
-	prev, ok, err := unstructured.NestedString(existingConfig, configPath...)
-	if err != nil {
-		return out, []error{err}
-	}
-	if ok {
-		if err := unstructured.SetNestedField(out, prev, configPath...); err != nil {
-			return out, []error{err}
-		}
-	}
+	servicesSubnetConfigPath := []string{"servicesSubnet"}
+	bindAddressConfigPath := []string{"servingInfo", "bindAddress"}
+	bindNetworkConfigPath := []string{"servingInfo", "bindNetwork"}
 
 	errs := []error{}
+
 	serviceCIDR, err := network.GetServiceCIDR(listers.NetworkLister, recorder)
 	if err != nil {
 		errs = append(errs, err)
+
+		// Pass through existingConfig
+		prev, ok, err2 := unstructured.NestedString(existingConfig, servicesSubnetConfigPath...)
```

Comment:
why err2? Use err.

Reply:
stomping on the `err` above seems like a recipe for, well, errors.

Reply:
err is the last command's error by convention. If you want to preserve one, rename it.</wednesday-bike-shedding>